pyre/pyrex/tests/fixtures/cpyext_callables.c

```
@@ -0,0 +1,202 @@
+/* The same, stating what was going on instead of naming an object. */
+static PyObject *report_unraisable_msg(PyObject *self, PyObject *unused)
+{
+    (void)self;
+    (void)unused;
+    PyErr_SetString(PyExc_ValueError, "boom");
+    PyErr_FormatUnraisable("Exception ignored while doing %s", "the thing");
+    Py_RETURN_NONE;
+}
+
+/* The version the extension is running against, and the interpreter it is in. */
+static PyObject *runtime_identity(PyObject *self, PyObject *unused)
```

Clang (14.0.6)

[warning] 2 adjacent parameters of 'runtime_identity' of similar type ('PyObject *') are easily swapped by mistake

(bugprone-easily-swappable-parameters)